quantum/quantum.c

```diff
@@ -201,6 +204,10 @@ bool process_record_quantum(keyrecord_t *record) {
   keypos_t key = record->event.key;
   uint16_t keycode;
 
+  #ifdef VELOCIKEY_ENABLE
+    if (velocikey_enabled()) velocikey_accelerate();
```

Comment:
The [Coding Conventions](https://docs.qmk.fm/#/contributing?id=coding-conventions) for QMK recommend: 
```suggestion
    if (velocikey_enabled()) { velocikey_accelerate(); }
```

Reply:
Done!